dlhub_sdk/client.py

```diff
@@ -397,6 +400,9 @@ def easy_publish(self, title: str, creators: Union[str, list[str]], short_name:
         model_info.set_creators(creators, affiliations or [])  # affiliations if provided, else empty list
         model_info.set_name(short_name)
 
+        if paper_doi is not None:
+            model_info.add_related_resource(paper_doi[0], "DOI", paper_doi[1])
```

Comment:
I like it, but how about we fix it to "isDescribedBy" to provide a shortcut for people just looking to associate a model with the paper that describes it?